modules/apps/transfer/keeper/msg_server_test.go

```diff
@@ -191,31 +121,31 @@ func (suite *KeeperTestSuite) TestMsgTransfer() {
 			path = ibctesting.NewTransferPath(suite.chainA, suite.chainB)
 			path.Setup()
 
-			coin1 = sdk.NewCoin(sdk.DefaultBondDenom, sdkmath.NewInt(100))
 			msg = types.NewMsgTransfer(
 				path.EndpointA.ChannelConfig.PortID,
 				path.EndpointA.ChannelID,
-				sdk.NewCoins(coin1),
+				testCoins,
```

Comment:
instead of modifying this for this test, maybe we can update [TestCoins](https://github.com/cosmos/ibc-go/blob/72e2b6b5be718e5501f6cc54acb39e314a87a92b/testing/values.go#L60) to have multiple entries and just be the default, so we can use either `TestCoin` or `TestCoins` depending on the need.